http-clients/src/main/java/com/palantir/remoting/http/FeignClientFactory.java

```diff
@@ -104,6 +106,8 @@ public static FeignClientFactory of(
                 .errorDecoder(errorDecoder)
                 .client(clientSupplier.apply(sslSocketFactory))
                 .options(options)
+                .logger(new Slf4jLogger(FeignClientFactory.class))
+                .logLevel(Level.BASIC)
```

Comment:
I assume you can override this using the standard mechanism provided by your slf4j implementation?